src/main/java/frc/robot/subsystems/Pneumatics.java

```diff
@@ -0,0 +1,43 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Pneumatics extends SubsystemBase {
+
+  private Compressor compressor = new Compressor(0);
+
+  private DoubleSolenoid hoodSolenoid = new DoubleSolenoid(2, 3);
```

Comment:
I think the solenoids for each subsystem should be in the subsystem.  Please move this, and the associated methods, to the shooter subsystem.